drivers/can/can_mcux_flexcan.c

```diff
@@ -966,6 +973,7 @@ static const struct can_driver_api mcux_flexcan_driver_api = {
 		.irq_config_func = mcux_flexcan_irq_config_##id,	\
 		.phy = DEVICE_DT_GET_OR_NULL(DT_INST_PHANDLE(id, phys)),\
 		.max_bitrate = DT_INST_CAN_TRANSCEIVER_MAX_BITRATE(id, 1000000), \
+		.irq_n = DT_INST_IRQ_BY_NAME(id, common, irq), \
```

Comment:
Beware that the number of IRQs used depends on the SoC. Not all use the `common` IRQ, so this will not work across all supported SoCs.

Reply:
I tried to fix it in the next commit 2e89435f761f1f619589efbaf3a2c302e2a3d4da. I suggest to list all interrupt names in `interrupt-names` property and generate two additional functions to enable and disable interrupts.

Reply:
Thanks. I suggest squashing the two commits into one.